iidm/iidm-api/src/main/java/com/powsybl/iidm/network/ShuntCompensatorLinearModel.java

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.network;
+
+/**
+ * @author Miora Ralambotiana <miora.ralambotiana at rte-france.com>
+ */
+public interface ShuntCompensatorLinearModel extends ShuntCompensatorModel {
+
+    /**
+     * Get the susceptance per section in S.
+     */
+    double getbPerSection();
```

Comment:
````getbPerSection```` or ````getBPerSection```` ?

Reply:
For consistence, we might as well rename as `getBPerSection`